README.md

```diff
@@ -19,8 +19,22 @@ This Cabal Git repository contains the following main packages:
 The canonical upstream repository is located at
 https://github.com/haskell/cabal.
 
-Ways to get the `cabal-install` binary
---------------------------------
+## Support window for releases
+
+| Latest release | 3.14 |
```

Comment:
From looking at the rendered version, the first line is treated as a header, which looks a little weird.

Reply:
It would make more sense if it actually stated the support windows and the difference between them. I'd suggest some text but I only have a vague idea of our actual support windows aside from the comments in `validate.yml` which I've seen too many times 😀 .

Reply:
Hm, also this raises a fairly big problem with LTS releases in our context: support for newer compilers, which (given the way we handle them) requires major version bumps. Do we need to add an escape hatch somewhere for when someone quite reasonably expects an LTS `cabal-install` to work with a newer `ghc`?